checkpoint_quantum_firewall/assets/logs/checkpoint-quantum-firewall.yaml

```diff
@@ -0,0 +1,946 @@
+    name: Layer Name
+    path: checkpoint.quantum.firewall.layer_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Malware Action
+    path: checkpoint.quantum.firewall.malware_action
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Malware Rule Name
+    path: checkpoint.quantum.firewall.malware_rule_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Matched Category
+    path: checkpoint.quantum.firewall.matched_category
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Operation
+    path: checkpoint.quantum.firewall.operation
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: OS Name
+    path: checkpoint.quantum.firewall.os_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Protection Name
+    path: checkpoint.quantum.firewall.protection_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Protection Type
+    path: checkpoint.quantum.firewall.protection_type
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Protocol
+    path: checkpoint.quantum.firewall.protocol
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Reject Category
+    path: checkpoint.quantum.firewall.reject_category
+    source: log
+    type: string
+  - facetType: range
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Server Inbound Packets
+    path: checkpoint.quantum.firewall.server_inbound_packets
+    source: log
+    type: integer
+  - facetType: range
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Server Outbound Packets
+    path: checkpoint.quantum.firewall.server_outbound_packets
+    source: log
+    type: integer
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Severity
+    path: checkpoint.quantum.firewall.severity
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: SmartDefense Profile
+    path: checkpoint.quantum.firewall.smartdefense_profile
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Source
+    path: checkpoint.quantum.firewall.src
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Source Machine Name
+    path: checkpoint.quantum.firewall.src_machine_name
+    source: log
+    type: string
+  - facetType: list
+    groups:
+      - Checkpoint Quantum Firewall
+    name: Status
```

Comment:
Can you add a sample where this attribute is used ? I'd like to understand if there's value in remapping this as the standard status attribute

Reply:
The value would be succeded and failed while we would add the status remapper to ensure the value get converted OK/Emergency but we would preserve this to ensure our filters in dashboard show the original value.

Reply:
It is OK to keep the original attribute but there's no need to create a facet for the attribute to be able to query it in dashboards.